java-sdk/src/main/java/com/netflix/conductor/sdk/workflow/executor/task/AnnotatedWorkerExecutor.java

```diff
@@ -187,10 +198,12 @@ public void startPolling() {
         }
 
         LOGGER.info("Starting workers with threadCount {}", workerToThreadCount);
+        LOGGER.info("Worker domains {}", workerDomains);
```

Comment:
Can it be combined into one log output?

Reply:
I want to keep this separate for the visibility and since its only printed once.